privacyidea/lib/eventhandler/responsemangler.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
```

Comment:
maybe we should add the ``handler`` suffix to the filename as well to make auto-discovery easier in the future.

Reply:
I dont think so.
I would discover by inherited classes and not by filenames.

Reply:
How do you discover by inherited classes?
Anyway, i'm ok with that.